ksqldb-engine/src/main/java/io/confluent/ksql/util/SharedKafkaStreamsRuntimeImpl.java

```diff
@@ -221,16 +221,13 @@ public void start(final QueryId queryId) {
     if (collocatedQueries.containsKey(queryId) && !collocatedQueries.get(queryId).everStarted) {
       if (!kafkaStreams.getTopologyByName(queryId.toString()).isPresent()) {
         try {
-          kafkaStreams.addNamedTopology(collocatedQueries.get(queryId).getTopology())
-              .all()
-              .get();
-        } catch (ExecutionException | InterruptedException e) {
-          final Throwable t = e.getCause() == null ? e : e.getCause();
+          kafkaStreams.addNamedTopology(collocatedQueries.get(queryId).getTopology());
+        } catch (final Exception e) {
```

Comment:
I don't think we want to wrap all exception here

Reply:
What kind of exceptions can be thrown here? It seems inside `addNamedTopology` we do not throw directly but always via the future object.